logic/nodes.go

```diff
@@ -444,6 +446,35 @@ func GetAllNodesAPI(nodes []models.Node) []models.ApiNode {
 	return apiNodes[:]
 }
 
+// DeleteExpiredNodes - goroutine which deletes nodes which are expired
+func DeleteExpiredNodes(ctx context.Context, peerUpdate chan *models.Node) {
+	for {
+		select {
+		case <-ctx.Done():
+			close(peerUpdate)
```

Comment:
we are already closing channel here, the program will panic closing channel twice
https://github.com/gravitl/netmaker/blob/d978cfe0c896760dab65d9999cc06c30181416cd/logic/zombie.go#L83